### PR TITLE
fix bug of refresh from disable refresh to refresh

### PR DESCRIPTION
**Pull request by Lijiaoa**

### CODE REVIEW

File: src/webui/src/App.tsx [hunk 1]
```diff
@@ -66,13 +67,12 @@ class App extends React.Component<{}, AppState> {
...
-    
+
     changeInterval = (interval: number): void => {
         this.setState({ interval });
-        if (this.timerId === null && interval !== 0) {
-            window.setTimeout(this.refresh);
-        } else if (this.timerId !== null && interval === 0) {
-            window.clearTimeout(this.timerId);
+        if (this.flag === false) {
+            this.refresh();
```
Comment (liuzhe-lz) [inline src/webui/src/App.tsx:74]: `refresh()` will setup a (sequence of) timer. If I spam changing interval, there will be bunch of timers.

File: src/webui/src/App.tsx [hunk 2]
```diff
@@ -17,7 +17,8 @@ interface AppState {
 }
 
 class App extends React.Component<{}, AppState> {
-    private timerId!: number | null;
+    private timerId!: number | undefined;
+    private flag: boolean = true;
```
Comment (liuzhe-lz) [inline src/webui/src/App.tsx:21]: Please use a meaningful name. I can't really understand what this flag does.

Comment (Lijiaoa) [reply src/webui/src/App.tsx:21]: OK

File: src/webui/src/App.tsx (hunk 1 above)
Comment (Lijiaoa) [reply src/webui/src/App.tsx]: had updated.

**Pull request merged**